kube-fluentd/README.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+
+This directory contains docker images with fluentd, which can be later configured to collect
+log from any source.
+
+# Usage
+
+Each image is built with its own set of plugins which you can later use in configuration. Set of
+plugin is enumerated in Gemfile. You can find details about fluentd configuration on the
+[official site](http://docs.fluentd.org/articles/config-file).
+
+In order to configure fluentd image, you should mount directory with `.conf` files to
+`/etc/fluent/config` or add files to this directory by building a new image on top. All `.conf`
```

Comment:
> Traditionally, directories with extra configuration files have the .d suffix, so how about /etc/fluentd/config.d?

ACK

> If users mount the config files over `/etc/fluentd/config`, they'll overwrite/hide whatever config you have there. It might be best to split configurations into the main config (in `/etc/fluent/fluent.conf` or whatever name you pick for your main config file), vendor config (e.g., in `/etc/fluentd/config`), and user config (e.g., in `/etc/fluentd/config/extra`).

As I mentioned somewhere below, I wanted to be able to control the output plugin for now, so it doesn't make sense to separate those two.

What you describe is actually achievable by building a new image on top of one of these, to infer some additional structure.